Add new PHP log INIs (#21851)
DataDog/documentation: content/en/tracing/trace_collection/library_config/php.md

`DD_TRACE_DEBUG`
: **INI**: `datadog.trace.debug`<br>
**Default**: `0`<br>

> Enable debug mode. When `1`, log messages are sent to the device or file set in the `error_log` INI setting. The actual value of `error_log` might be different than the output of `php -i` as it can be overwritten in the PHP-FPM/Apache configuration files. Takes precendence over `DD_TRACE_LOG_LEVEL` if active.

Reviewer: ```suggestion
Enable debug mode. When `1`, log messages are sent to the device or file set in the `error_log` INI setting. The actual value of `error_log` may be different than the output of `php -i` as it can be overwritten in the PHP-FPM/Apache configuration files. Takes precedence over `DD_TRACE_LOG_LEVEL` if active.
```

`DD_TRACE_LOG_LEVEL`
: **INI**: `datadog.trace.log_level`<br>
**Default**: `Error`<br>
Sets a precise log level. The log level follows RUST_LOG conventions; accepted log levels are `error`, `warn`, `info`, `debug`, `trace` and `off`.

`DD_TRACE_LOG_FILE`
: **INI**: `datadog.trace.log_file`<br>
**Default**: ``<br>

> Specifies a log file. If none is specified, logs go to the default php error location. Note that, to debug datadog-ipc-helper issues (e.g. submission of telemetry), the log file must be specified.

Reviewer: ```suggestion
Specifies a log file. If none is specified, logs go to the default PHP error location. To debug datadog-ipc-helper issues (for example, submission of telemetry), you must specify the log file.
```
